src/add.rb

```diff
@@ -0,0 +1,25 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name: 'Unknown', parent_permission: true)
+    @id = Random.rand(1..200)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    of_age? == true || @parent_permission == true
```

Comment:
- [ ] I think it would be better if you just called here the method to be of_age() ||  @parent_permission so this is don't have meaning here because it is method without calling and  instead of hard coding and write of_age? == true

```ruby
 def can_use_services? 
    is_of_age?() || @parent_permission
  end
```

Reply:
@Reem-lab It's ok, I didn't see it in that way, thanks for clarifying and giving me the hints. I appreciated it.

Reply:
Due to linter errors, I modified the of_age parentheses.
![image](https://user-images.githubusercontent.com/97189760/199137059-aaf0c469-0989-43d9-ba0f-ddcf1afe8734.png)